base/multimedia.jl

```diff
@@ -94,9 +94,12 @@ your images to be displayed on any PNG-capable `AbstractDisplay` (such as IJulia
 to `import Base.show` in order to add new methods to the built-in Julia function
 `show`.
 
-The default MIME type is `MIME"text/plain"`. There is a fallback definition for `text/plain`
-output that calls `show` with 2 arguments. Therefore, this case should be handled by
-defining a 2-argument `show(io::IO, x::MyType)` method.
+To customize human-readable text output for objects of type `T`, it is highly recommended
+to overload `show(::IO, ::MIME"text/plain", ::T)` rather than [`show(::IO, ::T)`](@ref show)
```

Comment:
I find it weird to say here that you shouldn't use the two-arg `show`, since we're in the docstring for the three-argument method. Anyway that's a repetition from the warning already printed in the docstring for the two-arg `show`, which will appear right before this one.